PWGHF/vertexingHF/AliAnalysisTaskSEXic0Semileptonic.cxx

```diff
@@ -35,6 +35,7 @@
 #include "AliAnalysisManager.h"
 #include "AliAODMCHeader.h"
 #include "AliAODHandler.h"
+#include "AliExternalTrackParam.h"
```

Comment:
Are these additional `include` useful for this task? If not I would suggest you to remove them, especially for the `AliAnalysisTaskSEHFTreeCreator` task, which does not belong to the vertexingHF library and hence causes the errors in the compilation